.github/workflows/e2e.yml

```diff
@@ -30,9 +30,9 @@ jobs:
           rustup default ${{ matrix.rust }}
           rustup component add rustfmt
       - name: Build
-        # note: since there is no 'cargo clean' here, the results of scripts/prepare-dfx-assets.sh
-        # are often cached.
-        run: cargo build --locked --release
+        run: |
+          cargo clean
```

Comment:
since we're running `cargo clean`, I think we can skip caching `target` directory
https://github.com/dfinity/sdk/pull/2513/files#diff-3e103440521ada06efd263ae09b259e5507e4b8f7408308dc227621ad9efa31eR25